dockernet/tests/integration_tests.bats

```diff
@@ -152,6 +152,34 @@ setup_file() {
   assert_equal "$diff" $STAKE_AMOUNT
 }
 
+@test "[INTEGRATION-BASIC-$CHAIN_NAME] packet forwarding automatically liquid stake and ibc transfer stAsset to original network" {
```

Comment:
Have you run these tests all the way through and confirmed they pass? I'm have a slight recollection of needing to bump down the original autopilot test so that it didn't interfere with the other tests

side note: we discussed in platform prioritization, that moving these tests to a different framework is one of the highest priority items for the beginning of next year. So hopefully this will be less painful moving soon!

Reply:
I think I did run them, but will run again after addressing comments!

Reply:
This comment is a bit stale, but I think this individual test succeeded, but it had caused an LSM test to fail.

It was resolved in a separate PR though!

Reply:
ah, thanks!